tests/plotting/test_plotting.py

```diff
@@ -1850,8 +1873,10 @@ def test_set_focus():
 @skip_no_plotting
 def test_set_viewup():
     plane = pyvista.Plane()
+    plane_higher = pyvista.Plane(center=(0, 0, 1), i_size=0.5, j_size=0.5)
     p = pyvista.Plotter()
-    p.add_mesh(plane, color="tan", show_edges=True)
+    p.add_mesh(plane, color="tan", show_edges=False)
+    p.add_mesh(plane_higher, color="red", show_edges=False)
     p.set_viewup((1.0, 1.0, 1.0))
     p.show(before_close_callback=verify_cache_image)
```

Comment:
Do you also see faint diagonal lines (which aren't technically edges) on planes when `show_edges=True`? I noticed that when I switche to the dev wheels.

But I see you changed to test to be a lot more visually differentiated, and thus hopefully more robust.

Reply:
> Do you also see faint diagonal lines (which aren't technically edges) on planes when show_edges=True? I noticed that when I switche to the dev wheels.

Yes, I see them too.  It's a visual artifact and an issue that should be raised on https://github.com/Kitware/VTK

Reply:
BTW, this was changed because line widths are different on vtk `9.0.1` vs `9.0.20210612`.  They're unnecessary to show as they do not indicate if your view is "up"; only two planes stacked on top of each other will do that.

Reply:
I see, thanks for the explanation. Makes a lot of sense.